examples/accordion/css/accordion.css

```diff
@@ -1,16 +1,26 @@
 .Accordion {
-    border: 1px solid hsl(0, 0%, 82%);
-    border-radius: .3em;
-    box-shadow: 0 1px 2px hsl(0, 0%, 82%);
+    margin: 0;
+    padding: 0;
+    border: 2px solid hsl(0, 0%, 82%);
+    border-radius: 7px;
+    width: 20em;
+}
+
+.Accordion.focus {
+    border-color: hsl(216, 94%, 73%);
 }
 
+.Accordion.focus h3 {
+    background-color: hsl(0, 0%, 97%);
+}
+
+
 .Accordion > * + * {
     border-top: 1px solid hsl(0, 0%, 82%);
 }
 
 .Accordion-trigger {
```

Comment:
Firefox is adding a dotted border around this element's `::-moz-focus-inner`, which looks rather busy since there's also a border around the child `.Accordion-title`:

![screen shot 2018-09-19 at 2 58 53 pm](https://user-images.githubusercontent.com/7939037/45775178-f1678680-bc1c-11e8-982e-5724f789174d.png)

Can be fixed with:
```css
.Accordion-trigger::-moz-focus-inner {
  border: none;
}
```